Overlord.Api/Program.cs

```diff
@@ -7,7 +7,7 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            var builder = WebApplication.CreateBuilder(args);
+            WebApplicationBuilder? builder = WebApplication.CreateBuilder(args);
```

Comment:
From a compiler perspective this generates the same code. 
It is also redundant and doesn't add to the readability since it is very obvious what the type is.